tsconfig.json

```diff
@@ -20,11 +20,5 @@
       "@styles": ["./src/components/BrandConfigProvider/styles/types"]
     }
   },
-  "include": [
-    "index.ts",
-    "src/**/*",
-    "lib/**/*",
-    "example/**/*",
```

Comment:
It looks this line was removed. What's the motivation there?

Reply:
I meant to not commit that, I could not run tests unless the example repo was setup and did not want to go down that rabbit hole, will revert.

Reply:
@nicolls1 is the intention to still remove this diff?